templates/windows-template.j2

```diff
@@ -54,7 +54,7 @@ objects:
         volumes:
           - name: volume-1
             persistentVolumeClaim:
-              claimName: claimName: vm-${NAME}-disk-01
+              claimName: vm-${NAME}-disk-01
```

Comment:
Not related?

Reply:
No and yes, the windows template will fail due to this, so I could make it a separate patch if you really want